lib/koans/19_genservers.ex

```diff
@@ -0,0 +1,137 @@
+defmodule GenServers do
+  use Koans
+
+  @intro "GenServers"
+
+  defmodule BicycleLock do
+    use GenServer
+
+    #####
+    # External API
+
+    def start_server(init_password) do
```

Comment:
I think it's a convension to keep the name of this function `start_link`. It makes your gen servers more "supervisors" friendly :-)